### ISSUE TITLE
append_if_no_line appends line always...

### ISSUE THREAD

**Issue by larryebaum** [sous-chefs/line#53]
The following code block:
```
append_if_no_line 'Set MULE_HOME' do
  path '/home/muleesb/.bash_profile'
  line 'MULE_HOME=/muleesb/mule-enterprise-standalone-3.7.2'
end
```

Should write the line to the file in path and not write the line if the line already exists. It writes the line on every converge.

**Comment by damacus** [comment]
Confirmed bug using the `append-if-no-line-ubuntu-1504` suite with new tests. 

Thanks for reporting this @larryebaum

**Comment by damacus** [comment]
Fun, this shouldn't actually be happening...

```
        * append_if_no_line[Operation 1] action edit
           - Updating file /tmp/dangerfile
         * append_if_no_line[Operation 2] action edit (up to date)
```

**Comment by larryebaum** [comment]
Verified incorrect behavior in Chef v12.21.3 as well as in Chef v13.1.31

**Comment by MarkGibbons** [comment]
f.puts new_resource.line if found

      unless found
        converge_by "Updating file #{new_resource.path}" do
          f.puts new_resource.line
        end
      end

Adds the new line if found or unless found.

**Comment by damacus** [comment]
Closed by #55
